src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff/DiffGeneratorFactory.cs

```diff
@@ -0,0 +1,112 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiSymbolExtensions;
+using Microsoft.DotNet.ApiSymbolExtensions.Logging;
+
+namespace Microsoft.DotNet.ApiDiff;
+
+public static class DiffGeneratorFactory
+{
+    /// <summary>
+    /// The default attributes to exclude from the diff.
+    /// </summary>
+    public static readonly string[] DefaultAttributesToExclude = [
```

Comment:
I think this can be improved by using the contents of this file by default: https://github.com/dotnet/runtime/blob/main/eng/DefaultGenApiDocIds.txt

Those are the attributes GenAPI excludes when generating the ref source files.

Reply:
Maybe for now, this string array is enough, and I can improve this in a follow-up PR.

Reply:
Created https://github.com/dotnet/sdk/issues/46922 to track it.